backend/src/test/resources/fhir/bundle.json

```diff
@@ -159,7 +159,7 @@
             "valueCodeableConcept": {
               "coding": [
                 {
-                  "system": "http://terminology.hl7.org/CodeSystem/v3-NullFlavor",
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0189",
```

Comment:
question out of curiosity does the version matter here? 🤔

Reply:
I don't know what the version number actually means since there is no "v2-NullFlavor".

Reply:
ahhh ok!